src/algokit/core/sandbox.py

```diff
@@ -46,9 +46,9 @@ def get_min_compose_version() -> str:
 
 
 class ComposeSandbox:
```

Comment:
There is logic to detect if any configuration is out of date. Do we need to change that to ignore the value of dev mode?